src/frontend/js/components/AddressesManagement/AddressForm.tsx

```diff
@@ -41,6 +42,31 @@ const AddressForm = ({ handleReset, onSubmit, address }: Props) => {
   const [languageCode] = intl.locale.split('-');
   const countryList = countries.getNames(languageCode);
 
+  const getLocalizedErrorMessage = (
+    error: Maybe<
+      | string
+      | {
+          key: ErrorKeys;
+          values: Record<PropertyKey, string | number | Array<string | number>>;
+        }
+    >,
+  ) => {
+    if (!error) return undefined;
+
+    let message;
+    let values;
+
+    if (typeof error === 'string' || errorMessages[error.key] === undefined) {
+      // If the error has not been translated we return a default error message.
+      message = errorMessages[ErrorKeys.MIXED_INVALID];
+    } else {
+      message = errorMessages[error.key];
+      values = error.values;
+    }
+
+    return intl.formatMessage(message, values);
```

Comment:
```suggestion
 
    if (typeof error === 'string' || errorMessages[error.key] === undefined) {
      // If the error has not been translated we return a default error message.
      return intl.formatMessage( errorMessages[ErrorKeys.MIXED_INVALID]);
    }
    return intl.formatMessage(errorMessages[error.key], error.values);
```
I'm still not mad of declaring a variable without setting it and using it after, at the end I find that confusing, wdyt of this rework to clarify